helm/m3db-operator/README.md

```diff
@@ -0,0 +1,5 @@
+### Helm Charts for M3 on Kubernetes
```

Comment:
Same comment here re: the fact that we only manage the dbnodes themselves so far and not query service / aggregator etc.

Reply:
ditto